packages/jest-cli/src/index.ts

```diff
@@ -6,20 +6,11 @@
  */
 
 // TODO: remove @jest/core exports for the next major
```

Comment:
next major, or kill them now?

Reply:
`jest-cli` is not really public API anyway, is it? So sooner rather than later I think